src/Jackett.Common/Indexers/IPTorrents.cs

```diff
@@ -190,53 +190,53 @@ protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuer
                     release.Comments = release.Guid;
 
-                    var descString = qRow.Find(".t_ctime").Text();
+                    var descString = row.QuerySelector(".t_ctime").TextContent;
                     var dateString = descString.Split('|').Last().Trim();
                     dateString = dateString.Split(new[] { " by " }, StringSplitOptions.None)[0];
                     release.PublishDate = DateTimeUtil.FromTimeAgo(dateString);
 
-                    var qLink = row.ChildElements.ElementAt(3).Cq().Children("a");
-                    release.Link = new Uri(SiteLink + WebUtility.UrlEncode(qLink.Attr("href").TrimStart('/')));
+                    var qLink = row.QuerySelector("a[href^=\"/download.php/\"]");
+                    release.Link = new Uri(SiteLink + WebUtility.UrlEncode(qLink.GetAttribute("href").TrimStart('/')));
 
-                    var sizeStr = row.ChildElements.ElementAt(5).Cq().Text();
+                    var sizeStr = row.Children[5].TextContent;
                     release.Size = ReleaseInfo.GetBytes(sizeStr);
 
-                    release.Seeders = ParseUtil.CoerceInt(qRow.Find(".t_seeders").Text().Trim());
-                    release.Peers = ParseUtil.CoerceInt(qRow.Find(".t_leechers").Text().Trim()) + release.Seeders;
+                    release.Seeders = ParseUtil.CoerceInt(row.QuerySelector(".t_seeders").TextContent.Trim());
+                    release.Peers = ParseUtil.CoerceInt(row.QuerySelector(".t_leechers").TextContent.Trim()) + release.Seeders;
 
-                    var catIcon = row.Cq().Find("td:eq(0) a");
-                    if (catIcon.Length >= 1) // Torrents - Category column == Icons
-                        release.Category = MapTrackerCatToNewznab(catIcon.First().Attr("href").Substring(1));
+                    var catIcon = row.QuerySelector("td:nth-of-type(1) a");
+                    if (catIcon != null) // Torrents - Category column == Icons
+                        release.Category = MapTrackerCatToNewznab(catIcon.GetAttribute("href").Substring(1));
                     else // Torrents - Category column == Text or Code
                         //release.Category = MapTrackerCatDescToNewznab(row.Cq().Find("td:eq(0)").Text()); // Works for "Text" but only contains the parent category
                         throw new Exception("Please go to " + SiteLink + "settings.php and change the \"Torrents - Category column\" option to \"Icons\". Wait a minute (cache) and then try again.");
 
-                    var filesElement = row.Cq().Find("a[href*=\"/files\"]"); // optional
-                    if (filesElement.Length == 1)
-                        release.Files = ParseUtil.CoerceLong(filesElement.Text());
+                    var filesElement = row.QuerySelector("a[href*=\"/files\"]"); // optional
+                    if (filesElement != null)
+                        release.Files = ParseUtil.CoerceLong(filesElement.TextContent);
```

Comment:
This one is looking for exactly one result, instead of >0 of results. This could be because more than one is an error and the filesElement shouldn't be used, or just an oversight of the original dev.

Reply:
Can't be more than 1. I think it's clear in this way.